cli/state.rs

```diff
@@ -268,76 +268,23 @@ impl ModuleLoader for State {
     Ok(module_specifier)
   }
 
-  /// Given an absolute url, load its source code.
   fn load(
     &self,
     module_specifier: &ModuleSpecifier,
     maybe_referrer: Option<ModuleSpecifier>,
-    is_dyn_import: bool,
+    _is_dyn_import: bool,
```

Comment:
I'm a bit confused about this `is_dyn_import` argument. What was its purpose before? Why is it not needed anymore?

Reply:
It was used for permission checks - it is still used, but in `prepare_load()` function (for the same purpose). 

With this PR calling `ModuleLoader.load()` results in loading already cached file; all of the permission check, etc. are done during building of module graph